contractcourt/chain_arbitrator.go

```diff
@@ -1010,7 +1010,8 @@ func (c *ChainArbitrator) NotifyContractUpdate(chanPoint wire.OutPoint,
 		return fmt.Errorf("can't find arbitrator for %v", chanPoint)
 	}
 
-	return arbitrator.notifyContractUpdate(update)
+	arbitrator.notifyContractUpdate(update)
```

Comment:
The commit body here should be populated with a message outlining the issue and the fix (copying the PR body should be sufficient).

Reply:
changed commit body